endpoints/IncomingEndpoint.ts

```diff
@@ -66,8 +72,9 @@ export class IncomingEndpoint extends ApiEndpoint {
                     await createDialogflowMessage(sessionId, read, modify, response, this.app);
                     await handlePayloadActions(this.app, read, modify, http, persistence, sessionId, vToken, response);
                 } catch (error) {
-                    this.app.getLogger().error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${error.message}`);
-                    throw new Error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${error.message}`);
+                    this.app.getLogger().error(`${Logs.DIALOGFLOW_REST_API_ERROR} ${getError(error).message}`);
```

Comment:
`getError` function returns string. Here you are trying to access the `message` property of the string which throws an error. Didn't it break in your testing 🙂?

Reply:
@Shailesh351 Oh good catch, it always returns the String. So I shall not use ".message" property.
Instead I should use the whole error value.

I really wonder why it didn't throw any error during my testing.